Migration and update seeds.

pedrosino opened the pull request.

pedrosino commented:
Screenshot:
![events](https://cloud.githubusercontent.com/assets/1881933/21487807/a4af184c-cbbb-11e6-83bc-6b530e21aef4.png)

jfly commented on WcaOnRails/db/seeds/events.seeds.rb:
```diff
@@ -1,23 +1,23 @@
-('333ft', 'Rubik''s Cube: With feet', 80, 'time', '3x3 with feet'),
+('444', '4x4x4 Cube', 30, 'time', '4x4x4 Cube'),
+('555', '5x5x5 Cube', 40, 'time', '5x5x5 Cube'),
+('222', '2x2x2 Cube', 20, 'time', '2x2x2 Cube'),
```
Could you reorder these by rank?

jfly commented on WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb:
```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+class ChangeEventOrderAndNames < ActiveRecord::Migration
+  def up
+    # No change for Rubik's Cube event.
```
Just to make it clear, how about we do one line at  the end where we set `cellName = name`?

pedrosino replied:
Ah, that's smart.

pedrosino commented:
I have fixed @jfly comments. I also pushed this to staging and the dropdown menu looks good:
![eventsdrop](https://cloud.githubusercontent.com/assets/1881933/21499931/4eebc944-cc21-11e6-9bca-26ce3bef124b.png)

Since rubocop was complaining about the test line being too long (251x245) I changed Rubocop's policy to 255. Let me know if there's a better way of dealing with this.

jfly commented on WcaOnRails/db/migrate/20161226223701_change_event_order_and_names.rb:
```diff
@@ -0,0 +1,65 @@
+    SQL
+
+    execute <<-SQL
+      update Events set cellName = name;
```
Looks good. Could you create an issue to drop the `cellName` column and assign it to yourself? I don't want to take care of it in this PR, but I also would like to take care of it sooner than later.

pedrosino replied:
#1054

jfly commented on WcaOnRails/spec/controllers/competitions_controller_spec.rb:
```diff
@@ -597,8 +597,8 @@
         post = assigns(:post)
-        expect(post.body).to include "World records: Jeremy Fleischman 3x3 one-handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3 fewest moves 25 (single), 3x3 fewest moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2 Cube 10.00 (single)"
-        expect(post.body).to include "North American records: Jeremy Fleischman 3x3 one-handed 41.00 (single), 3x3 one-handed 40.00 (single)"
+        expect(post.body).to include "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), 3x3x3 Fewest Moves 26.00 (average), Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
```
So this is our longest line of Ruby code? I'd say it's time to split it up into multiple lines. I like to write multiline strings like this:

```ruby
expect(post.body).to include(
  "World records: Jeremy Fleischman 3x3x3 One Handed 50.00 (average), " +
  "Vincent Sheu (2006SHEU01) 3x3x3 Fewest Moves 25 (single), " +
  "3x3x3 Fewest Moves 26.00 (average), " +
  "Vincent Sheu (2006SHEU02) 2x2x2 Cube 10.00 (single)"
)
```

pedrosino replied:
Rubocop says we should use \ instead of +.

jfly commented:
I'm trusting you to check that our seeds match the migration. In the future, you could avoid this worry by having the migration drop the events table and load from seeds.

LGTM.
